crates/dbsp/src/trace/spine_async/mod.rs

```diff
@@ -548,6 +558,9 @@ where
             .filter_map(|(level, slot)| slot.try_start_merge(level).map(|batches| (level, batches)))
             .collect::<Vec<_>>();
         for (level, batches) in start_merges {
+            gauge!(ONGOING_MERGES_PER_LEVEL, "worker" => Runtime::worker_index_str(), "level" => LEVELS_AS_STR[level], "id" => ident)
+                .set(batches.len() as f64);
```

Comment:
It looks like we update ONGOING_MERGES_PER_LEVEL only when we initiate a merge. For it to be up-to-date, I think that we should also update it (to 0) when we complete a merge (just above, around the time we call `merge_complete`, presumably).